docs/content/latest/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -15,29 +15,242 @@ isTocNested: true
+| TServer TCMalloc Stats               | In Use (Heap Memory Usage): the number of bytes used by the application. Typically, this does not match the memory use reported by the OS because it does not include TCMalloc overhead or memory fragmentation.<br>Total (Reserved Heap Memory): the number of bytes of the system memory reserved by TCMalloc.<br> | This metric is informational and should not be subject to alerting. You may consider this information while examining alerts on other metrics.<br><br>Also consider the following:<br>Free Heap Memory: the number of bytes in free, mapped pages in a page heap. These bytes can be used to fulfill allocation requests. They always count towards virtual memory usage, and unless the underlying memory is swapped out by the OS, they also count towards physical memory usage.<br>Unmapped Heap Memory: the number of bytes in free, unmapped pages in a page heap. These are bytes that have been released back to the OS, possibly by one of the MallocExtension Release calls. They can be used to fulfill allocation requests, but typically incur a page fault. They always count towards virtual memory usage, and depending on the OS, usually do not count towards physical memory usage.<br>Thread Cache Memory Limit: a limit to the amount of memory that TCMalloc dedicates for small objects. In some cases, larger numbers trade off more memory use for improved efficiency.<br>Thread Cache Memory Usage: a measure of some of the memory TCMalloc is using (for small objects). | ![img](/images/yp/metrics39.png) |
+| Log / WAL Stats / Node               | Log bytes read: the size (in bytes) of reads from the WAL since the tablet start.<br>Log cache size: the total per-tablet size of consensus entries kept in memory. The log cache attempts to keep all entries which have not yet been replicated to all followers in memory, but if the total size of those entries exceeds the configured limit within an individual tablet, the oldest is evicted. | If the log cache size is greater than zero, the followers are behind. You should issueing an alert if the value spikes or remains above zero for an extended period of time. | ![img](/images/yp/metrics40.png) |
+| Log / WAL Cache Num Ops / Node       | The number of times the log cache is accessed.               | Outside of an xCluster replication deployment, a number greater than zero means some of the followers are behind. | ![img](/images/yp/metrics42.png) |
+| GLog messages                        | The following log levels are available:<br><br>Info: the number of INFO-level log messages emitted by the application.<br>Warning: Number of WARNING-level log messages emitted by the application.<br>Error: the number of ERROR-level log messages emitted by the application. | It is recommended to use a log aggregation tool for log analysis. You should review the ERROR-level log entries on a regular basis. | ![img](/images/yp/metrics43.png) |
```

Comment:
"Number of" - other blocks are "the number of"

Reply:
fixed